### PR TITLE
PORIJATE-X-back-link-to-waste-search

### PR DESCRIPTION
**Pull request by khalima**
To test: 
- Checkout this branch and do `drush cc all`
- Go to a waste type taxonomy term page (similar to this one: https://pori.dev.wunder.io/jate/energiajate) and see that there is a "Back to waste search" link after the description

### CODE REVIEW

File: drupal/code/themes/custom/kada/preprocess/taxonomy_term.preprocess.inc [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?php
+
+function kada_preprocess_taxonomy_term(&$variables) {
+  if (
```
Comment (demoshane) [inline drupal/code/themes/custom/kada/preprocess/taxonomy_term.preprocess.inc]: https://www.drupal.org/docs/develop/standards/coding-standards#controlstruct coding standards for control structure

File: drupal/code/themes/custom/kada/src/scss/component/misc/_waste-search.scss [hunk 2]
```diff
@@ -597,3 +597,20 @@
     }
   }
 }
+
+a.button--waste {
```
Comment (demoshane) [inline drupal/code/themes/custom/kada/src/scss/component/misc/_waste-search.scss]: Should be probably under
a {
  &.button--waste {
  }
}

**Comment by khalima** [comment]
Should be fixed now. Pre-commit hook + linters would help quite a lot in these cases..

**Comment by demoshane** [comment]
Approved.

**Pull request merged**